### PR TITLE
💄 Document block redesign block redesign #640

### LINKED ISSUE #640: :lipstick:  Document Block Redesign: block redesign
## User stories

As a user, I want the "Document Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/document-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/document-message-block/`.

### PR DESCRIPTION
**Pull request by Mutagah**
# Description

This PR makes UI changes to document block redesign.

Fixes #640

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

document_block_1

document_block_2

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Same changes as those requested on the phone block

File: libs/features/convs-mgr/stories/blocks/library/document-message-block/src/lib/components/document-block/document-block.component.ts [hunk 1]
```diff
@@ -25,19 +27,28 @@ export class DocumentBlockComponent implements OnInit, OnDestroy {
   whatsappLimit: boolean;
   messengerLimit: boolean;
 
+  svgIcon = ''
+
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/document-message-block/src/lib/components/document-block/document-block.component.ts:31]: should be `svgIcon!: string;` or `svgIcon: string;`

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/document-block.jsplumb.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function _DocumentBlockDecoratePlumb(block: DocumentMessageBlock, comp: C
     target: true,
     cssClass:"block_endpoint",
     endpoint: "Dot",
-    anchor: "Left",
+    anchor: [0, 0.11 , 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/document-block.jsplumb.ts:25]: lets leave this as left for now.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Almost done just sync your changes and resolve conflicts

File: libs/features/convs-mgr/stories/blocks/library/document-message-block/src/lib/components/document-block/document-block.component.ts (hunk 1 above)
Comment (Mutagah) [reply libs/features/convs-mgr/stories/blocks/library/document-message-block/src/lib/components/document-block/document-block.component.ts]: > should be `svgIcon!: string;` or `svgIcon: string;`

Changes requested have been made.

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/document-block.jsplumb.ts (hunk 2 above)
Comment (Mutagah) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/document-block.jsplumb.ts]: > lets leave this as left for now.

Changes requested have been addressed.

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**